tsl/src/continuous_aggs/options.c

```diff
@@ -284,3 +286,26 @@ continuous_agg_update_options(ContinuousAgg *agg, WithClauseResult *with_clause_
 		elog(ERROR, "cannot alter finalized option for continuous aggregates");
 	}
 }
+
+void
+warnIfHierarchicalRealtimeCagg(int32 parent_mat_hypertable_id, bool materialized_only)
```

Comment:
I think it would be better to use lowercase and underscore-separated naming for this function because it would be more consistent with the current style

Reply:
sure; renamed the method to `warn_if_hierarchical_realtime_cagg`